common/src/models/service.rs

```diff
@@ -121,31 +122,140 @@ Most recent deploys for {}
+            output.push(get_static_folder_table(static_folders));
+        };
+
+        if let Some(persist) = resource_groups.get("Persist") {
+            output.push(get_persist_table(persist));
+        };
+
+        output.join("\n")
+    }
+}
+
+fn get_databases_table(databases: &Vec<&resource::Response>) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_content_arrangement(ContentArrangement::DynamicFullWidth)
+        .set_header(vec![
+            Cell::new("Type").set_alignment(CellAlignment::Center),
+            Cell::new("Connection string").set_alignment(CellAlignment::Center),
+        ]);
+
+    for database in databases {
+        let info = serde_json::from_value::<DatabaseReadyInfo>(database.data.clone()).unwrap();
+
+        table.add_row(vec![
+            database.r#type.to_string(),
+            info.connection_string_public(),
+        ]);
+    }
+
+    format!(
+        r#"These databases are linked to this service
 {table}
 "#,
-        )
+    )
+}
+
+fn get_secrets_table(secrets: &[&resource::Response]) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_header(vec![Cell::new("Key").set_alignment(CellAlignment::Center)]);
+
+    let secrets = serde_json::from_value::<SecretStore>(secrets[0].data.clone()).unwrap();
+
+    for key in secrets.secrets.keys() {
+        table.add_row(vec![key]);
     }
+
+    format!(
+        r#"These secrets can be accessed by the service
+{table}
+"#,
+    )
+}
+
+fn get_static_folder_table(static_folders: &[&resource::Response]) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_content_arrangement(ContentArrangement::DynamicFullWidth)
+        .set_header(vec![
+            Cell::new("Static Folders").set_alignment(CellAlignment::Center)
+        ]);
+
+    for folder in static_folders {
+        let path = serde_json::from_value::<PathBuf>(folder.data.clone())
+            .unwrap()
+            .display()
+            .to_string();
+
+        table.add_row(vec![path]);
+    }
+
+    format!(
+        r#"These static folders can be accessed by the service
+{table}
+"#,
+    )
+}
+
+fn get_persist_table(persist_instances: &[&resource::Response]) -> String {
+    let mut table = Table::new();
+
+    table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_content_arrangement(ContentArrangement::DynamicFullWidth)
+        .set_header(vec![
+            Cell::new("Persist Instances").set_alignment(CellAlignment::Center)
+        ]);
+
+    for _ in persist_instances {
+        table.add_row(vec!["Instance"]);
+    }
+
+    format!(
+        r#"These instances are linked to this service
+{table}
+"#,
+    )
 }
```

Comment:
This one I'm not so sure about how we should handle, since the output just contains the project name. :thinking: